asaplib/__init__.py

```diff
@@ -1 +1 @@
-__all__ = ['cluster', 'io', 'kde', 'kernel', 'pca', 'plot', 'fit', 'compressor', 'util']
```

Comment:
As an aside I think with all future PRs it would be good to ensure that all of the examples run correctly before merging.

Reply:
Can we automate this with pytest somehow? Then they could all be ran on travis for example for each commit and/or pull

Reply:
Yep, that would be ideal; I was thinking the same. I guess setting up Travis or Jenkins might be worth the effort if we foresee the usage of the package expanding significantly

Reply:
Can you help setting up Travis or Jenkins? I've no experience with them